src/Microsoft.Health.Development.IdentityProvider/Microsoft.Health.Development.IdentityProvider.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Description>Identity primitives used by Microsoft Health.</Description>
```

Comment:
Maybe something like `Identity primitives used by Microsoft Health for local development scenarios.`

Reply:
Sounds good!